### PR TITLE
RAD-352 Add accordion option to radiologyOrderDetails portlet

### PR DESCRIPTION
**Pull request by Schwasrz94**
## Description

* add box header and box to radiologyOrderDetails.jsp
* add accordion to radiologyOrderDetails.jsp if withAccordion parameter
  is set

## Related Issue

see https://issues.openmrs.org/browse/RAD-352

### CODE REVIEW

File: omod/src/main/webapp/portlets/radiologyOrderDetails.jsp [hunk 1]
```diff
@@ -1,7 +1,28 @@
 <%@ include file="/WEB-INF/template/include.jsp"%>
```
Comment (teleivo) [inline omod/src/main/webapp/portlets/radiologyOrderDetails.jsp]: @Schwarz94 please add a comment header like other portlets have that describe the param (name, optional, description...)

Comment (teleivo) [inline omod/src/main/webapp/portlets/radiologyOrderDetails.jsp]: @Schwasrz94 please update PR

File: omod/src/main/webapp/portlets/radiologyOrderDetails.jsp [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
 
+<%--
+  available properties:
+    orderUuid -> (String) the uuid of the radiology order to display
+                          if not set or no radiology order was found nothing will be displayed
+    withAccordion -> if set the radiology order details box will be shown as an accordion 
+--%>
+
```
Comment (Schwasrz94) [inline omod/src/main/webapp/portlets/radiologyOrderDetails.jsp]: Isn't that what you asked for?

Comment (teleivo) [inline omod/src/main/webapp/portlets/radiologyOrderDetails.jsp]: @Schwasrz94 yes missed it. withAccordion is optional and boolean right? If so it should say so

Comment (Schwasrz94) [inline omod/src/main/webapp/portlets/radiologyOrderDetails.jsp]: ok. no, it could be any value.

**Pull request merged**